sdk/go/x/auto/stack.go

```diff
@@ -751,6 +830,8 @@ func (s *Stack) runPulumiCmdSync(
 	args ...string,
 ) (string, string, int, error) {
 	var env []string
+	debugEnv := fmt.Sprintf("%s=%s", "PULUMI_DEBUG_COMMANDS", "true")
```

Comment:
Was this intended?

Reply:
Yeah, the `--event-log` flag is only accessible if the debug env var is present. We discussed removing that requirement but have decided to keep it for now because we will (probably at some point) implement similar functionality via a `--json` flag and didn't want to derail this work by conflating the two.